steps/bitrise-step-nowsecure-auto-analysis/0.1.0/step.yml

```diff
@@ -0,0 +1,54 @@
+title: NowSecure Auto Analysis
+summary: |
+  Performs security analysis using NowSecure Auto.
+description: |
+  Performs security analysis using NowSecure Auto.
+website: https://github.com/sp3nx0r/bitrise-step-nowsecure-auto-analysis
+source_code_url: https://github.com/sp3nx0r/bitrise-step-nowsecure-auto-analysis
+support_url: https://github.com/sp3nx0r/bitrise-step-nowsecure-auto-analysis/issues
+published_at: 2020-01-17T12:54:54.250963064-06:00
+source:
+  git: https://github.com/sp3nx0r/bitrise-step-nowsecure-auto-analysis.git
+  commit: fa6a0e20bf8c53102e3f2e1a6939291dcad93be1
+host_os_tags:
+- osx-10.10
+- ubuntu-16.04
+type_tags:
+- test
+toolkit:
+  bash:
+    entry_file: step.sh
+deps:
+  brew:
+  - name: curl
+  apt_get:
+  - name: curl
+is_requires_admin_user: false
+is_always_run: false
+is_skippable: false
+run_if: ""
+inputs:
+- opts:
+    description: |
+      This usually gets generated by the the "Gradle Runner" or "XCode Archive & Export" step
+    is_required: true
+    summary: The app file you want to upload to BrowserStack, usually $BITRISE\_APK\_PATH
+      or $BITRISE\_IPA\_PATH
+    title: A Bitrise generated APK or IPA path
+  upload_path: $BITRISE_APK_PATH
```

Comment:
Is it a better default value than `$BITRISE_IPA_PATH`? If no, it is better to leave the input empty, so the workflow editor UI will highlight that a required input is not specified.